services/src/api/src/main/java/org/openkilda/northbound/dto/links/LinkPropsDto.java

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.northbound.dto.links;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Data
+@EqualsAndHashCode(exclude = "props")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+public class LinkPropsDto {
+
+    private static final String DEFAULT = "";
```

Comment:
What's the purpose of the constant which is applied to different by meaning fields? I.e. switches and ports.

Or is it DEFAULT_PROPERTY_VALUE?

Reply:
When switch/port is not specified, it will use empty string which will stand for 'any switch'/'any port'. This is actually copy pasted from the original DTO class in NB module

Reply:
fixed to match the new pull request. I had an old class